src/sudo/pam.rs

```diff
@@ -5,14 +5,32 @@ use crate::log::{dev_info, user_warn};
 use crate::pam::{PamContext, PamError, PamErrorType, PamResult};
 use crate::system::term::current_tty_name;
 
+pub(super) struct InitPamArgs<'a> {
```

Comment:
Will Rust ever get named function arguments? :-)

Reply:
Maybe. Maybe not.

Reply:
The syntax would be pretty obvious to me:

```rust
pub fn something { arg: &str } -> SomeType {
    ...
}
```
```rust
  something { arg: "hello" };
```
Alas, this PR is not the place to discuss that I guess 😛